### ISSUE TITLE
[a11y] Table structure - Courses

### ISSUE THREAD

**Issue by tdy** [cs-education/classTranscribe#64]
Courses URL: https://classtranscribe.ncsa.illinois.edu/courses

Data cells should have corresponding headers. This supports screen readers and other assistive technologies in programmatically parsing and conveying table structure to users (see [WCAG 2.0 SC 1.3.1 Info and Relationships][1]).

Visually the table is fine, but the hidden elements throw off the cell/header alignment. Currently there are 2 hidden columns but only 1 hidden header. The 1st hidden column has a hidden “Term” header. The 2nd hidden column (with codes like 3aa47947-7afa-411f-adcf-155818d54338) is missing a hidden header which offsets the remaining headers by 1 (so the final “Watch” column is effectively missing a header).

[1]: https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html
